data/saas/config/oracle_responsys_config.yml

```diff
@@ -75,3 +82,20 @@ saas_config:
                 - dataset: <instance_fides_key>
                   field: profile_list_recipient.riid
                   direction: from
+    - name: profile_extension_recipient
+      requests:
+        read:
+          - request_override: oracle_responsys_profile_extension_recipients_read
+            param_values:
+              - name: placeholder_email
+                identity: email
```

Comment:
We can remove this, we're already linking to another collection via the `profile_list` and `responsys_id` references and it doesn't look like we're using the email in the `oracle_responsys_profile_extension_recipients_read` request override

Reply:
removed.